build/musicbrainz-dev/Dockerfile

```diff
@@ -0,0 +1,78 @@
+FROM metabrainz/base-image
+LABEL Maintainer="yvanzo@metabrainz.org"
+
+ENV DOCKERIZE_VERSION v0.5.0
```

Comment:
```suggestion
ENV DOCKERIZE_VERSION=v0.5.0
```
for consistency :)

Reply:
Consistency with?

Reply:
with line 64 to 70 of this file.

Reply:
That is consistent with `build/musicbrainz/Dockerfile`. We don’t want these to differ by coding style.